dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -186,6 +198,71 @@ export default defineComponent({
         : getCountryNameFromCountryCode(kpiValue as string) ?? kpiValue;
     },
 
+    /**
+     * Generates a list of readible strings (or just a single one) combining suppliers and their associated countries
+     * @param numberOfSuppliersPerCountryCode the map of number of suppliers and associated companies
+     * from which strings are written
+     * @returns the constructed collection of readable strings
+     */
+    generateReadableCombinationOfNumberOfSuppliersAndCountries(
+      numberOfSuppliersPerCountryCode?: Map<string, number | undefined | null>
+    ) {
+      if (numberOfSuppliersPerCountryCode != undefined) {
+        const readableListOfSuppliersAndCountries = Array.from(numberOfSuppliersPerCountryCode.entries()).map(
+          ([countryCode, numberOfSuppliers]) => {
+            const countryName = getCountryNameFromCountryCode(countryCode) ?? countryCode;
```

Comment:
I think we don't need the "`??" operator, because `getCountryNameFromCountryCode()` always returns a string.

Reply:
Possible. Wouldn't it be safer to keep it. Just in case it catches a bug we aren't aware of?

Reply:
I don't think it makes sense to keep it, because it will never be executed anyways.

If the countryCode passed to `getCountryNameFromCountryCode()` is somehow wrong, it will return an empty string.
Therefore the `?? countryCode` will actually never be executed.

Reply:
Same goes for the `?? countryCodeShort` in line 196 of this file.

Reply:
Example:
`"" ?? "n/a";` will return `""` and not `"n/a"`

Reply:
👍